### PR TITLE
Sharethis

### PR DESCRIPTION
**Pull request by astrofrog**
Another routine pull request - this time for sharing buttons (bottom of the front page only).

http://astrofrog.github.com/astropy-website/

I've added Facebook, Twitter, and Google+ via ShareThis. Any reasons not to do this?

I will leave this pull request up for a day before merging, in case there are any objections. This is mostly for esthetics, as the functionality may need tweaking once the website is live (can't check all the buttons properly until then).

### CODE REVIEW

**Comment by embray** [comment]
Cool! I'm happy with the placement--it's unobtrusive, but not completely off the radar. I assume that once the actual website is updated the preview image will be the new logo?

File: _themes/bootstrap/layout.html [hunk 1]
```diff
@@ -115,6 +127,13 @@
       {% trans sphinx_version=sphinx_version|e %}Created using <a href="http://sphinx.pocoo.org/">Sphinx</a> {{ sphinx_version }}.{% endtrans %}<br/>
     {%- endif %}
     </p>
+    {% if title == "Welcome" %}
+    <p align='center'>
+      <span class='st_facebook_large' displayText='Facebook' st_title='The Astropy Project' st_url='http://www.astropy.org' st_summary='The Astropy project is a common effort to develop a single core package for Astronomy.'></span>
+      <span class='st_googleplus_large' displayText='Google +1' st_title='The Astropy Project' st_url='http://www.astropy.org' st_summary='The Astropy project is a common effort to develop a single core package for Astronomy.'></span>
+      <span class='st_twitter_large' displayText='Tweet' st_title='The Astropy Project: a common effort to develop a single core Python package for Astronomy!' st_url='http://www.astropy.org'</span>
```
Comment (embray) [inline _themes/bootstrap/layout.html]: This span tag is missing an end bracket.

**Comment by astrofrog** [comment]
@iguananaut - for facebook sharing, I think it _should_ be - facebook looks at the opengraph tags if they are there (in theory), and should use the image from there. But I can't test this till we go live... The issue is that the sharing URLs don't match this preview.

**Comment by embray** [comment]
Yeah. For facebook it's using the old logo. But I figured that was just because you were using the real URL :)

**Comment by taldcroft** [comment]
@astrofrog - Looks good to me.

File: _themes/bootstrap/layout.html (hunk 1 above)
Comment (astrofrog) [inline _themes/bootstrap/layout.html]: Fixed

File: _themes/bootstrap/layout.html [hunk 2]
```diff
@@ -115,6 +127,13 @@
       {% trans sphinx_version=sphinx_version|e %}Created using <a href="http://sphinx.pocoo.org/">Sphinx</a> {{ sphinx_version }}.{% endtrans %}<br/>
     {%- endif %}
     </p>
+    {% if title == "Welcome" %}
+    <p align='center'>
+      <span class='st_facebook_large' displayText='Facebook' st_title='The Astropy Project' st_url='http://www.astropy.org' st_summary='The Astropy project is a common effort to develop a single core package for Astronomy.'></span>
+      <span class='st_googleplus_large' displayText='Google +1' st_title='The Astropy Project' st_url='http://www.astropy.org' st_summary='The Astropy project is a common effort to develop a single core package for Astronomy.'></span>
+      <span class='st_twitter_large' displayText='Tweet' st_title='The Astropy Project: a common effort to develop a single core Python package for Astronomy!' st_url='http://www.astropy.org'></span>
+    </p>
+    {% endif %}
```
Comment (embray) [inline _themes/bootstrap/layout.html:136]: As an HTML pedant it bugs me that Sharethis insists on using non-valid element attributes.  But that's something I'll have to just pester them about :)

Comment (astrofrog) [inline _themes/bootstrap/layout.html:136]: If you find a better/alternative solution to ShareThis, please do feel free to open a pull request - I just did what was easiest for now.

**Pull request merged**